Mathlib/Order/Filter/Basic.lean

```diff
@@ -848,6 +848,15 @@ theorem Eventually.choice {r : α → β → Prop} {l : Filter α} [l.NeBot] (h
   choose! f hf using fun x (hx : ∃ y, r x y) => hx
   exact ⟨f, h.mono hf⟩
 
+lemma skolem {ι : Type*} {α : ι → Type*} [∀ i, Nonempty (α i)]
+    {P : ∀ (i : ι), α i → Prop} {F : Filter ι} :
+    (∃ b : Π i, α i, ∀ᶠ i in F, P i (b i)) ↔ ∀ᶠ i in F, ∃ b, P i b := by
```

Comment:
This would be less confusing for me:
```suggestion
lemma skolem {ι : Type*} {α : ι → Type*} [∀ i, Nonempty (α i)]
    {P : ∀ i : ι, α i → Prop} {F : Filter ι} :
    (∃ b : (Π i, α i), ∀ᶠ i in F, P i (b i)) ↔ ∀ᶠ i in F, ∃ b, P i b := by
```
For consistency with [Classical.skolem](https://leanprover-community.github.io/mathlib4_docs/Init/Classical.html#Classical.skolem), please switch the LHS and the RHS.